src/random_test.cpp

```diff
@@ -1314,6 +1343,9 @@ void create_database_tablespace(Thd1 *thd) {
   if (!no_tbs)
     g_tablespace = {"innodb_system", "tab02k", "tab04k"};
 
+  for (int i=1; i<=options->at(Option::CREATE_UNDO_TABLESPACE)->getInt(); i++) {
```

Comment:
Is create_undo_tablespaces the number of undo table-spaces or probability of executing create_undo_tablespace.
Seems like you are using same variable for both. Please fix that.

Reply:
Fixed